docs/docs-content/release-notes.md

```diff
@@ -11,6 +11,33 @@ sidebar_custom_props:
 tags: ["release-notes"]
 ---
 
+## November 30, 2023 - Release 4.1.12
+
+This release contains minor bug fixes.
+
+### Bug Fixes
+
+
+- An issue with downloading the backup manifest has been updated to allow the backup manifest to be downloaded by the host cluster successfully every 1-minute interval.
```

Comment:
```suggestion
- An issue with downloading the backup manifest has been resolved. Palette now allows the backup manifest to be downloaded by the host cluster at 1-minute intervals.
```
- I feel like we should not say "an issue has been updated", though I also understand if we do not want two sentences in a single bullet.
- I do think that "successfully" and "every" are redundant here with "allow" and "interval" already present.